graphs_sections_A-I.R

```diff
@@ -906,20 +906,20 @@ plot_webchart(filepath="Income and employment/IE-worklessness-chart2", chart_typ
 plot_webchart(filepath="Injuries/inj_road_traffic_chart_1", chart_type = "multiline",
            xvar="class2", yvar="measure", group="class1", pal_col=palnogroups,
            title="Chart 1. Road traffic casualties, Scotland",
-           sourc="<a href='https://www.transport.gov.scot/publication/reported-road-casualties-scotland-2017/'>Transport Scotland</a>", 
-           xaxtitle="Year", yaxtitle="Number")
+           sourc="<a href='https://www.transport.gov.scot/publication/reported-road-casualties-scotland-2019/statistical-tables-1/'>Transport Scotland</a>. Injury severity definitions changed in 2019.", 
```

Comment:
I would expand the note slightly to say that therefore 2019 figures are not comparable with the rest of the trend. If it fits!